src/domains/citizen-auth-app/01_network.tf

```diff
@@ -69,6 +69,12 @@ data "azurerm_subnet" "app_backend_l2_snet" {
   resource_group_name  = local.vnet_common_resource_group_name
 }
 
+data "azurerm_subnet" "ioweb_profile_snet" {
+  name                 = format("%s-ioweb-profile-snet", local.product)
```

Comment:
location `weu` is missing here

Reply:
fixed in https://github.com/pagopa/io-infra/pull/679/commits/68d692316214b8cd97fc28da3195bdf7f818de2c